bears/js/JSStandardBear.py

```diff
@@ -0,0 +1,25 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='standard',
+        output_format='regex',
+        output_regex=r'  (?P<filename>[^:]+):(?P<line>\d+):(?P<column>\d+):'
```

Comment:
Is the space after the quote intentional in `r'  (?P<filename>` ? Maybe you could capture it with `\s*` or similar ?